reorganize the table rate docs (#587)
pynucastro/pynucastro: docs/source/data_sources.rst

Tabulated Rates
---------------

> For electron captures and beta-decays, we used tabulated rates.  Tabular rates for reactions of the form :math:`\rm{A \rightarrow B}`.

Reviewer: ```suggestion
For electron captures and beta-decays (which are of the form :math:`\rm{A \rightarrow B}`), we use tabulated rates.
```
